build.cake

```diff
@@ -127,13 +128,28 @@ Task("Build")
 });
 
 Task("Pack")
+    .WithCriteria(() => !isPullRequest)
```

Comment:
Pack should also work on PRs. Publishing of resulting artifacts is prevent by appveyor config.

Reply:
There is a difference between publishing to nuget and placing it on the AppVeyor feed, the first is prevented the second is not.

I agree with Jan, if you have a lot of PR's you don't want artifacts in the project feed to appear on every PR. If there is a setting which prevents the artifact from being placed on the feed, I didn't look into this, than I would also remove the criteria. The package would than be available on the artifacts page of the job.

Reply:
> There is a difference between publishing to nuget and placing it on the AppVeyor feed, the first is prevented the second is not.

There is no auto publish for nuget here, so it prevents the second.
So WithCriteria here is not needed.

Documentation from appveyor:
>   disable_publish_on_pr: true     # disable publishing of .nupkg artifacts to
                                  # account/project feeds for pull request builds

Reply:
You convinced me, I removed the criteria.